parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoWriteSupport.java

```diff
@@ -337,6 +337,12 @@ private void writeAllFields(MessageOrBuilder pb) {
         List<FieldDescriptor> fieldDescriptors = messageDescriptor.getFields();
         for (FieldDescriptor fieldDescriptor : fieldDescriptors) {
           FieldDescriptor.Type type = fieldDescriptor.getType();
+
+          //For a field in a oneOf that isn't set don't write anything
+          if (fieldDescriptor.getContainingOneof() != null && !pb.hasField(fieldDescriptor)) {
+            continue;
+          }
```

Comment:
Maybe this approach is too simplistic, but it does seem like exactly the behavior needed.

Reply:
What about the read side?  Is there going to be any issue there?  Currently a value is always written, so there is a value to read.  Now that value is missing.

Reply:
@belugabehr That's a great question.  Everything seems ok when I tested it out locally. I wrote some data out to a parquet file with this change, read the data back into protobuf messages, and then wrote the data back to a second parquet file.  The two parquet files were identical, so everything round tripped successfully.  

I am struggling a bit to write unit tests for this case, but that's probably me getting up the parquet API learning curve.

Reply:
To be clear, I will try to add tests for this to my PR.